Milvus.Client/ArrayFieldData.cs

```diff
@@ -0,0 +1,187 @@
+                    arrayArray.Data.Add(new ScalarField {IntData = int8Data});
+                    break;
+
+                case MilvusDataType.Int16:
+                    Grpc.IntArray int16Data = new();
+                    var shorts = array as IEnumerable<short> ?? Enumerable.Empty<short>();
+                    int16Data.Data.AddRange(shorts.Select(x=>(int)x));
+                    arrayArray.Data.Add(new ScalarField {IntData = int16Data});
+                    break;
+
+                case MilvusDataType.Int32:
+                    Grpc.IntArray int32Data = new();
+                    int32Data.Data.AddRange(array as IEnumerable<int>);
+                    arrayArray.Data.Add(new ScalarField {IntData = int32Data});
+                    break;
+
+                case MilvusDataType.Int64:
+                    Grpc.LongArray int64Data = new();
+                    int64Data.Data.AddRange(array as IEnumerable<long>);
+                    arrayArray.Data.Add(new ScalarField {LongData = int64Data});
+                    break;
+
+                case MilvusDataType.Float:
+                    Grpc.FloatArray floatData = new();
+                    floatData.Data.AddRange(array as IEnumerable<float>);
+                    arrayArray.Data.Add(new ScalarField {FloatData = floatData});
+                    break;
+
+                case MilvusDataType.Double:
+                    Grpc.DoubleArray doubleData = new();
+                    doubleData.Data.AddRange(array as IEnumerable<double>);
+                    arrayArray.Data.Add(new ScalarField {DoubleData = doubleData});
+                    break;
+
+                case MilvusDataType.String:
+                    Grpc.StringArray stringData = new();
+                    stringData.Data.AddRange(array as IEnumerable<string>);
+                    arrayArray.Data.Add(new ScalarField {StringData = stringData});
+                    break;
+
+                case MilvusDataType.VarChar:
+                    Grpc.StringArray varcharData = new();
+                    varcharData.Data.AddRange(array as IEnumerable<string>);
+                    arrayArray.Data.Add(new ScalarField {StringData = varcharData});
+                    break;
+
+                case MilvusDataType.Json:
+                    Grpc.JSONArray jsonData = new();
+                    var enumerable = array as IEnumerable<string> ?? Enumerable.Empty<string>();
+                    jsonData.Data.AddRange(enumerable.Select(ByteString.CopyFromUtf8));
+                    arrayArray.Data.Add(new ScalarField {JsonData = jsonData});
+                    break;
+                case MilvusDataType.None:
+                    throw new MilvusException($"ElementType Error:{DataType}");
+                default:
+                    throw new MilvusException($"ElementType Error:{DataType}, not supported");
+            }
+        }
+
+        return fieldData;
+
+        /*
```

Comment:
Commented-out code accidentally left here? Should be removed?

Reply:
Sorry :)

Reply:
No problem :)